core/src/main/java/org/jruby/RubySignal.java

```diff
@@ -42,13 +42,13 @@
 
 @JRubyModule(name="Signal")
 public class RubySignal {
-    private final static SignalFacade SIGNALS = getSignalFacade();
 
-    private final static SignalFacade getSignalFacade() {
+    private final static SignalFacade SIGNAL_FACADE = initSignalFacade();
+
+    private final static SignalFacade initSignalFacade() {
         try {
-            Class realFacadeClass = Class.forName("org.jruby.util.SunSignalFacade");
-            return (SignalFacade)realFacadeClass.newInstance();
-        } catch(Throwable e) {
+            return org.jruby.util.SunSignalFacade.class.newInstance();
```

Comment:
somehow ended up hitting this `Class.forName` spot (at least) twice while 'naively' profiling.
likely a false alarm, but I still did review the class and its signal.rb pieces ...